src/scripts/main.js

```diff
@@ -1,5 +1,37 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', e => {
-  // write code here
+  if (e.target.closest('.wall')) {
```

Comment:
Better just to return the function if it doesn't match the condition. It helps to avoid nesting
```
if (!e.target.closest('.wall')) return;
... your code